tutorials/create-esxi-cluster-dedibox/configure-centos.mdx

```diff
@@ -0,0 +1,98 @@
+---
```

Comment:
Is this doc meant to be in this PR?

Reply:
It is there because of "failover IP" - I don't know why it is entirely green